source/controllers/include/controllers/impedance/Dissipative.hpp

```diff
@@ -243,33 +243,53 @@ Eigen::MatrixXd Dissipative<S>::compute_orthonormal_basis(const Eigen::MatrixXd&
 
 template<class S>
 void Dissipative<S>::compute_damping(const Eigen::VectorXd& desired_velocity) {
+  double tolerance = 1e-4;
   Eigen::MatrixXd updated_basis = Eigen::MatrixXd::Zero(this->nb_dimensions_, this->nb_dimensions_);
   switch (this->computational_space_) {
-    case ComputationalSpaceType::LINEAR:
+    case ComputationalSpaceType::LINEAR: {
+      Eigen::Vector3d linear_velocity = desired_velocity.segment(0, 3);
+      // only update the damping if the commanded linear velocity is non null
+      if (linear_velocity.norm() < tolerance) { return; }
```

Comment:
As usual my preference (and style guide) is to forbid inline bracket cases like this, for readability - the following makes the early return stand out much more prominently, which is the intention

```c++
if (linear_velocity.norm() < tolerance) {
  return;
}
```

Reply:
make sense. I have just corrected that